Reset the submodule download system avoiding git-submodule

filiatra opened the pull request.

mmoelle1 commented:
I always wanted to rename `gsUnum` and the flag `UNUM` into `gsUniversal` and `UNIVERSAL`. This might be the right time to do it. Can you do this, please?

filiatra commented:
> I always wanted to rename `gsUnum` and the flag `UNUM` into `gsUniversal` and `UNIVERSAL`. This might be the right time to do it. Can you do this, please?

OK I will

mmoelle1 reviewed (commented):
Could you please rename exprtk_unum_posit into exprtk_universal_posit?

mmoelle1 committed:
Gitlab ci (interruptable)

filiatra committed:
ci

mmoelle1 committed:
Gitlab ci (Docker)

mmoelle1 committed:
Merge branch 'remove_submodules' of https://github.com/gismo/gismo into remove_submodules

filiatra committed:
add git clean

filiatra committed:
add git clean

filiatra committed:
ci

filiatra committed:
ci

mmoelle1 committed:
Removed GISMO_WITH_ONURBS left-overs

filiatra committed:
ci

filiatra committed:
print output

mmoelle1 committed:
Updated docker creation scripts and README

mmoelle1 committed:
Removed leftover GISMO_WITH_...

mmoelle1 committed:
Merge branch 'remove_submodules' of https://github.com/gismo/gismo into remove_submodules

filiatra committed:
improve computeBoundary

hverhelst committed:
add small error message for setup, Clamped

hverhelst committed:
initial commit gsAlmostC1; hotfix in DPatch and small additions to core classes to support gsAlmostC1

hverhelst committed:
small

weinmueller committed:
add pybind stuff

weinmueller committed:
finished merge

weinmueller committed:
improve python files

hverhelst committed:
Merge remote-tracking branch 'origin/stable' into msplines_shell

hverhelst committed:
hotfix

weinmueller committed:
merge stable into remove_submodules

weinmueller committed:
fix paraview

filiatra committed:
merge stable

filiatra committed:
hotfix for element expr

filiatra committed:
small fix in gdExprEvaluator, add edge members in gsMesh

filiatra committed:
compilation fix

filiatra committed:
fix unittest

filiatra committed:
add flag for t-junction, for ruture implementation

filiatra committed:
fi for msvc2022

filiatra committed:
reduce computation time for bhiharmonic example

filiatra committed:
fix unittest

filiatra committed:
fix unittest

hverhelst committed:
add component to corners
add corners to dirichlet values

filiatra committed:
cleanup and a fix for computeTopology

hverhelst committed:
small

hverhelst committed:
small

hverhelst committed:
more small changes

hverhelst committed:
fix bug

filiatra committed:
fix compilation for cpp17

filiatra committed:
add few assertions, update readmes

filiatra committed:
add few assertions, update readmes

hverhelst committed:
add boundary option to getEVs

weinmueller committed:
change binding

hverhelst committed:
small change box top

hverhelst committed:
Merge branch 'remove_submodules_pascal' of github.com:gismo/gismo into remove_submodules_pascal

weinmueller committed:
add geometries surfaces

hverhelst committed:
add offset to matchWith

weinmueller committed:
finish merged

weinmueller committed:
add laplace

hverhelst committed:
added weak clamping

weinmueller committed:
moved surface example

weinmueller committed:
update gitignore

hverhelst committed:
fix for plotting two function sets

hverhelst committed:
fix support

roeltielen24 committed:
Adding p-multigrid method  (#473)

Co-authored-by: Angelos Mantzaflaris <angelos.mantzaflaris@inria.fr>
Co-authored-by: roeltielen <roeltielen@hotmail.com>
Co-authored-by: Stefan Takacs <stefan.takacs@numa.uni-linz.ac.at>

hverhelst committed:
small

hverhelst committed:
max_expr

hverhelst committed:
Merge remote-tracking branch 'origin/expression_small' into msplines_shell

hverhelst committed:
small

hverhelst committed:
Merge remote-tracking branch 'origin/stable' into submodule_gsUnstructuredSplines2

hverhelst committed:
Merge remote-tracking branch 'origin/msplines_shell' into submodule_gsUnstructuredSplines2

filiatra committed:
typos

filiatra committed:
improve paraview mesh plotting

filiatra committed:
improve closest point computations

filiatra committed:
add mesh2

filiatra committed:
add mesh2

filiatra committed:
cleanup

filiatra committed:
cleanup

filiatra committed:
add normal

filiatra committed:
add normal

filiatra committed:
fix for openmp

filiatra committed:
fix for openmp

filiatra committed:
Merge remote-tracking branch 'origin/stable' into ccsurf

filiatra committed:
cc surf

filiatra committed:
cc surf

filiatra committed:
add acc3

filiatra committed:
add acc3

filiatra committed:
rm old example

filiatra committed:
rm old example

filiatra committed:
cleanup

filiatra committed:
cleanup

filiatra committed:
minors

filiatra committed:
minors

filiatra committed:
improve dist computations

filiatra committed:
minor

filiatra committed:
Merge remote-tracking branch 'origin/stable' into ccsurf

filiatra committed:
fix eval flags

filiatra committed:
Merge remote-tracking branch 'origin/stable' into ccsurf

filiatra committed:
fix bug with flags

filiatra committed:
Merge remote-tracking branch 'origin/stable' into ccsurf

filiatra committed:
cleanup

filiatra committed:
cleanup

filiatra committed:
add off files

filiatra committed:
add off files

weinmueller committed:
add gitignore

weinmueller committed:
merge stable

weinmueller committed:
fix filedata

weinmueller committed:
python binding fix

weinmueller committed:
Revert "python binding fix"

This reverts commit b5cfe8f6b6afe81beae114a8c944147cc6404083.

weinmueller committed:
Revert "fix filedata"

This reverts commit e2337aed237d7c53bc6708f2d0f9c0d34dfe877e.

weinmueller committed:
Revert "merge stable"

This reverts commit 9db6c1773551cacd7792f501fa691330f25513e5, reversing
changes made to e9e76a8329c13179df3c88e4190cb99f466c9a7c.

filiatra committed:
bug fix in aux computations

hverhelst committed:
revert multipatch commit

weinmueller committed:
fix pybind function

filiatra committed:
fixes for Python

filiatra committed:
add constructor

filiatra committed:
Update gsFileData.hpp, etc for gsSurfMesh reading

filiatra committed:
Update gsFileData.hpp, etc for gsSurfMesh reading

filiatra committed:
Merge remote-tracking branch 'origin/stable' into ccsurf

filiatra committed:
add cc funcs

filiatra committed:
add cc funcs

filiatra committed:
add CC member functions

filiatra committed:
add CC member functions

filiatra committed:
small change

filiatra committed:
small change

Commit:
fixed MKL

Commit:
fixed MKL

weinmueller committed:
merge submodules_gsUnstructuredSplines2

weinmueller committed:
fix python bug

weinmueller committed:
add area function to FeElement

filiatra committed:
updates

filiatra committed:
updates

filiatra committed:
upd

filiatra committed:
upd

filiatra committed:
a

filiatra committed:
cc

filiatra committed:
add vtk

filiatra committed:
gsView

filiatra committed:
merge stable

filiatra committed:
fix merge

filiatra committed:
Merge remote-tracking branch 'origin/stable' into submodule_gsUnstructuredSplines

filiatra committed:
fix merge: add modified files from stable

filiatra committed:
a

filiatra committed:
Merge remote-tracking branch 'origin/stable' into remove_submodules

filiatra committed:
Merge remote-tracking branch 'origin/stable' into ccsurf

filiatra committed:
Add program for cc basis

filiatra committed:
fix def

weinmueller committed:
fix pybind error

filiatra committed:
small fix

hverhelst committed:
Merge remote-tracking branch 'origin/stable' into submodule_gsUnstructuredSplines

hverhelst committed:
Fixed some compilation errors in the gsDirichletValues (don't know why?)
Changed gsFunction->gsFunctionSet in gsBoundaryConditions

hverhelst committed:
small

hverhelst committed:
small fixes

hverhelst committed:
stable merge

hverhelst committed:
Merge remote-tracking branch 'origin/stable' into submodule_gsUnstructuredSplines

filiatra committed:
rm cpp

hverhelst committed:
small undos for merge

hverhelst committed:
small

hverhelst committed:
Merge remote-tracking branch 'origin/remove_submodules' into submodule_gsUnstructuredSplines

hverhelst committed:
gsWriteParaview precision

hverhelst committed:
Update scordelis_lo_roof.xml

hverhelst committed:
add simple

Commit:
Merge remote-tracking branch 'origin/stable' into ccsurf

filiatra committed:
Merge remote-tracking branch 'origin/stable' into ccsurf

Commit:
quadsplit code

Commit:
quad split update

filiatra committed:
0Merge remote-tracking branch 'origin/stable' into ccsurf

filiatra committed:
fitting mp

Commit:
updates

Commit:
Merge branch 'ccsurf' of https://github.com/gismo/gismo into ccsurf

Commit:
fitting multipatch splines

Commit:
multipatch extension

Commit:
multipatch fitting

Commit:
updates

Commit:
updates

Commit:
updates

Commit:
updates

Commit:
updates

Commit:
updates

Commit:
updates

Commit:
updates

Commit:
updates

filiatra committed:
partial fix

Commit:
updates

Commit:
updates

Commit:
updates

filiatra committed:
fitting

Commit:
updates

Commit:
updates

Commit:
updates

filiatra committed:
small fix

Commit:
updates

filiatra committed:
fix acc3

filiatra committed:
small fix in quad split

Commit:
updates and new file

hverhelst committed:
add directional refinement

Commit:
multipatch fitting code

Commit:
multipatch setting

Commit:
updates

Commit:
updates

Commit:
examples

filiatra committed:
improve second der

filiatra committed:
Merge remote-tracking branch 'origin/stable' into ccsurf

hverhelst committed:
add neon-side geometries (fitted)

hverhelst committed:
add degree 3 geom

hverhelst committed:
add neon files

hverhelst committed:
add smoothened geometries

hverhelst committed:
rename files

hverhelst committed:
add degreeIncrease and degreeDecrease
add matchWith with offsets to gsTensorBasis

hverhelst committed:
move neon files to gsUnstructuredSplines

hverhelst committed:
add plotInterfaces code
fix components in Clamped setup

hverhelst committed:
fix BCs get

hverhelst committed:
reverts

hverhelst committed:
gsView, biharmonic2_example, pMultiGrid_example

hverhelst committed:
Delete plotInterfaces.cpp

hverhelst committed:
revert gsMesh and gsFileData

hverhelst committed:
revert gsDirichletValues

hverhelst committed:
opennurbs and poisson2_example.dox reverts

hverhelst committed:
opennurbs and exprevaluator revert

hverhelst committed:
revert gsExprEvaluator gsDebug gsForwardDeclarations

hverhelst committed:
revert gsMultiPatch

hverhelst committed:
reverts

hverhelst committed:
revert gsExprEvaluator

hverhelst committed:
revert filedata

hverhelst committed:
pybind fix

hverhelst committed:
revert gsRemapInterface

hverhelst committed:
pybind fix

hverhelst committed:
Merge pull request #535 from gismo/submodule_gsUnstructuredSplines

Submodule gs unstructured splines

hverhelst committed:
Merge remote-tracking branch 'origin/stable' into remove_submodules

hverhelst committed:
Merge remote-tracking branch 'origin/stable' into remove_submodules

hverhelst committed:
rm gsOptProblem from IpOpt

hverhelst committed:
fix pi

Commit:
point clouds

filiatra committed:
merge stable

hverhelst committed:
add neon_side object file

Commit:
file

filiatra committed:
remove duplicate

filiatra committed:
xcMerge remote-tracking branch 'origin/stable' into remove_submodules

filiatra committed:
add gismo_require_submodule

filiatra committed:
fixes ipopt module

filiatra committed:
add ctr

filiatra committed:
clean cmake

filiatra committed:
changed GISMO_SUBMODULES variable typeto string, now appears in options

hverhelst committed:
fix boundary condition corner patch index
remove unsigned from gsMappedSpline
add functionAtCorner in gsMappedSingleBasis

filiatra committed:
merge

filiatra committed:
update fitting

filiatra committed:
Merge remote-tracking branch 'origin' into ccsurf

filiatra committed:
cleanup

filiatra committed:
merge ccsurf

filiatra committed:
resolve conflict

filiatra committed:
resolve conflict

filiatra committed:
warning fix

filiatra committed:
fix uniformrefine

filiatra committed:
rm scratch examples

filiatra committed:
renaming options

filiatra committed:
mv parasolid files

filiatra committed:
slight renaming
